### DISCUSSION TITLE
Support for stream and defer directive in graphql

### DISCUSSION THREAD

**Discussion by bhushanibm** [99designs/gqlgen#2251] [Ideas]
### What happened?
When will the stream and defer directive be supported ? Is there any discussion on supporting the directive ?
Also, is there any document which will list external people from supporting the above directive and steps that needs to be followed ?

### versions
 - `go run github.com/99designs/gqlgen version`?
 - `go version`?

**Comment by genert** [comment]
Defer is supported already. See example test on it https://github.com/99designs/gqlgen/blob/15c05a78f8590f5bc8ca193e4a82e410c6f4b8d5/codegen/testserver/singlefile/defer_test.go

**Comment by johanrd** [comment]
[@apollo/client@4.1.0](https://github.com/apollographql/apollo-client/releases/tag/%40apollo%2Fclient%404.1.0) just [added support for the @stream directive](https://github.com/apollographql/apollo-client/pull/12918). This is a big step for the maturity of the spec. Has anyone given it a try with gqlgen?
